akka-persistence/src/main/scala/akka/persistence/Persistence.scala

```diff
@@ -404,7 +404,7 @@ class Persistence(val system: ExtendedActorSystem) extends Extension {
     log.debug(s"Create plugin: $pluginActorName $pluginClassName")
     val pluginClass = system.dynamicAccess.getClassFor[Any](pluginClassName).get
     val pluginDispatcherId = pluginConfig.getString("plugin-dispatcher")
-    val pluginActorArgs = try {
+    val pluginActorArgs: List[AnyRef] = try {
```

Comment:
what was the warning here?

Reply:
Inferring the type to `List[Object]` ("is probably not what you want")